### PR TITLE
Hotfix play version

### PR DESCRIPTION
**Pull request by benshaw**
## Description
* Update Play and Client Versions
* Remove inheritance for Json

## Why
* Dependies need to be updated so logging can be pulled in
* updating play led to a bug due to the way the json formats where being made available, use composition over inheritance

## Type
- [ ] Bug Fix
- [ ] Feature Addition
- [ ] Refactor
- [X] HotFix

## Checklist
- [X] tested locally
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

### CODE REVIEW

**Comment by benshaw** [comment]
@ty before merging in the changes to dev i had
` Total 282, Failed 5, Errors 1, Passed 276 `
After I had
` Total 263, Failed 4, Errors 3, Passed 256`
Im assuming someting went ary in the merge from dev, can you confirm what numbers i should be seeing ?

File: project/Dependencies.scala [hunk 1]
```diff
@@ -55,8 +55,8 @@ object Dependencies {
       val cache       = "com.typesafe.play" %% "play-cache"        % version
       val test        = "com.typesafe.play" %% "play-test"         % version
       val jdbc        = "com.typesafe.play" %% "play-jdbc"         % version
-      val json        = "com.typesafe.play" %% "play-json"         % "2.6.14"
-      val jsonJoda    = "com.typesafe.play" %% "play-json-joda"    % "2.6.14"
+      val json        = "com.typesafe.play" %% "play-json"         % "2.9.1"
+      val jsonJoda    = "com.typesafe.play" %% "play-json-joda"    % "2.9.1"
       val mailer      = "com.typesafe.play" %% "play-mailer"       % "6.0.1"
       val mailerGuice = "com.typesafe.play" %% "play-mailer-guice" % "6.0.1"
```
Comment (xavji) [inline project/Dependencies.scala:61]: we have already gone through that exercise, and an upgrade of Play needs to come with its companion libraries like the mailer here or Silhouette...

Comment (benshaw) [reply project/Dependencies.scala:61]: what excercise ?
i think mailer is straightforward but when i attempted to upgrade sillhouette alot of things broke

Comment (benshaw) [reply project/Dependencies.scala:61]: my original attempt was to upgrade everything but it led to alot of errors

**Pull request closed without merging**